lib/constants/validators.dart

```diff
@@ -1,18 +1,18 @@
-String pwdValidator(String value) {
+Function(String) pwdValidator = (String value) {
   if (value.length < 8) {
     return 'Password must be longer than 8 characters';
   } else {
     return null;
   }
-}
+};
 
-String emailValidator(String value) {
+Function(String) emailValidator = (String value) {
   Pattern pattern =
-      r'^(([^<>()[\]\\.,;:\s@\"]+(\.[^<>()[\]\\.,;:\s@\"]+)*)|(\".+\"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$';
+      r'^[A-Za-z0-9._%+-]+@iiitsonepat.ac.in$';
```

Comment:
Regex takes in any alphanumeric followed by iiitsonepat.ac.in so its a wrong regex